datacentral.py

```diff
@@ -160,31 +164,50 @@ def process_datapackage(pkg_name, repo_dir):
     '''
     pkg_dir = os.path.join(repo_dir, pkg_name)
     pkg_info = {}
-    metadata = json.loads(open(os.path.join(pkg_dir, "datapackage.json")).read())
+    try:
+        metadata = json.loads(open(os.path.join(pkg_dir, "datapackage.json")).read())
+    except IOError:
+        raise ParseException("datapackage.json not found")
 
     # get main attributes
     pkg_info['name'] = pkg_name
+    pkg_info['homepage'] = repo_url
     pkg_info['original_name'] = metadata['name']
     pkg_info['title'] = metadata['title']
     pkg_info['license'] = metadata.get('license')
-    pkg_info['description'] = metadata['description']
-    pkg_info['sources'] = metadata.get('sources')
+    if pkg_info['license'] and 'title' in pkg_info['license']:
+        pkg_info['license'] = pkg_info['license']['title']
+    if not 'description' in metadata:
+        pkg_info['description'] = ""
+    else:
+        pkg_info['description'] = metadata['description']
```

Comment:
Instead of this if/else block you could simply write:

```python
pkg_info['description'] = metadata.get('description', '')
```
The `get` method on a dict takes care if a key doesn't exists and then uses the default value (`''` in this case).